fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -142,6 +142,7 @@
      * @param relPath the given path
      * @return the child of this resource
      */
+    @Deprecated
```

Comment:
I'd vote to just remove it and not deprecate it, either that or at least return one child because the method is totally useless otherwise.

Reply:
When I say its useless, I mean because it is just returning `null` right now

Reply:
I went ahead and removed it